test/e2e-common.sh

```diff
@@ -61,8 +61,11 @@ LATEST_SERVING_RELEASE_VERSION=$(latest_version)
 
 # Latest net-istio release.
 LATEST_NET_ISTIO_RELEASE_VERSION=$(
-  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | grep '"tag_name"' \
-    | cut -f2 -d: | sed "s/[^v0-9.]//g" | sort | tail -n1)
+  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | \
+  jq -r '.[].tag_name' | \
+  cut -f2 -d'-' | \
+  sort -V | \
+  tail -n1)
```

Comment:
That'll produce output `v1.0.0`. However, the actual tag is `knative-v1.0.0`, e.g. in release URL https://github.com/knative-sandbox/net-istio/releases/download/knative-v1.0.0/istio.yaml.

Or there's going to be `v1.0.0` as well?

Reply:
It really depends for what the version is used. If only for the download URL, then should use the full tag name, without stripping. If the version is also used in a different context, then maybe v1.0.0 is correct.

What's about introducing an addition `LATEST_NET_ISTIO_RELEASE_TAG` env var ?